components/eamxx/src/physics/p3/impl/p3_conservation_impl.hpp

```diff
@@ -28,7 +35,13 @@ ::cloud_water_conservation(const Spack& qc, const Scalar dt,
     qc2qr_autoconv_tend.set(enforce_conservation, qc2qr_autoconv_tend*ratio);
     qc2qr_accret_tend.set(enforce_conservation, qc2qr_accret_tend*ratio);
     qc2qi_collect_tend.set(enforce_conservation, qc2qi_collect_tend*ratio);
-    qc2qi_hetero_freeze_tend.set(enforce_conservation, qc2qi_hetero_freeze_tend*ratio);
+    if(use_hetfrz_classnuc){
+         qcheti_cnt.set(enforce_conservation, qcheti_cnt*ratio);
```

Comment:
Why are these named differently? Are these meant to correspond to the concentration tendencies like the rest of this block?

Reply:
If I understand your question correctly, these variables store quantities to compute heterogeneous ice formation using CNT. I followed the naming convention and logic used in SCREAMv0 P3 and E3SMv3 source codes. Please let me know if I misunderstood your questions.